source/_components/atome.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: "Atome Linky Sensor"
+description: "Instructions on how to integrate Atome Linky consumption data within Home Assistant."
+logo: enedis.png
+ha_release: 0.97
+ha_category:
+  - Energy
+  - Sensor
+ha_iot_class: Cloud Polling
+redirect_from:
```

Comment:
Please remove the `redirect_from`, this is only for existing pages

Reply:
Thx for reviewing, doing